samples/hybrid-connections/java/simple-http-demo/src/samples/HttpListener.java

```diff
@@ -0,0 +1,45 @@
+package samples;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+
+import com.microsoft.azure.relay.HybridConnectionListener;
+import com.microsoft.azure.relay.RelayedHttpListenerContext;
+import com.microsoft.azure.relay.RelayedHttpListenerResponse;
+import com.microsoft.azure.relay.StringUtil;
```

Comment:
>StringUtil [](start = 33, length = 10)

I don't love this name, "StringUtil", however I'm guessing it exists in this form in the SB or EH library.